library/network/src/modules/CWMFirewallInterfaces.rb

```diff
@@ -833,7 +831,7 @@ def CreateOpenFirewallWidget(settings)
         return { "widget" => :custom, "custom_widget" => not_installed_widget, "help" => "" }
       end
 
-      if services.any? { |s| !firewalld.api.service_supported?(s) }
+      if services.any? { |s| !firewalld.current_service_names.include?(s) }
```

Comment:
we already know the list of available service, so we avoid to call the api as it is slow.